java/ql/lib/ext/io.netty.handler.codec.http.cookie.model.yml

```diff
@@ -0,0 +1,11 @@
+extensions:
+- addsTo:
+    pack: codeql/java-all
+    extensible: summaryModel
+  data:
+    - ["io.netty.handler.codec.http.cookie", "Cookie", True, "domain", "()", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
+    - ["io.netty.handler.codec.http.cookie", "Cookie", True, "name", "()", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
+    - ["io.netty.handler.codec.http.cookie", "Cookie", True, "path", "()", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
+    - ["io.netty.handler.codec.http.cookie", "Cookie", True, "value", "()", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
```

Comment:
Currently the various setter methods for the http-related types aren't yet modeled, since there aren't sink models which would be the main reason that those setter methods are useful; and it seems less common for them to be used in the middle of a flow path.

Reply:
Sinks seem a bit more difficult to model - most of the output goes through methods like `ChannelOutboundInvoker.write`, but modelling that as, say, an XSS sink may lead to FPs when it's used for something else (like non-HTTP stuff). 

Though it's probably still a good idea to model the setters now so that if we do find a good way to model sinks then they'd work.